cfgov/preprocessed/js/molecules/Expandable.js

```diff
@@ -0,0 +1,251 @@
+    var $cueClose = $this.find( '.expandable_cue-close' ).not( $this.find( '.expandable .expandable_cue-close' ) );
+    var $content = $this.find( '.expandable_content' ).not( $this.find( '.expandable .expandable_content' ) );
+
+    $cueOpen.attr( 'style', '' );
+    $cueClose.attr( 'style', '' );
+    $content.attr( 'style', '' );
+
+  }
+  // ==========================================================================
+
+
+  // The Expandable context will directly be the Expandable when used in an ExpandableGroup.
+  var _dom = context.classList.contains( 'expandable' ) ?
+             context : context.querySelector( '.expandable' );
+  var _target = _dom.querySelector( '.expandable_target' );
+  var _content = _dom.querySelector( '.expandable_content' );
+  var _link = _dom.querySelector( '.expandable_link' );
+  var _cueOpen = _link.querySelector( '.expandable_cue-open' );
+  var _cueClose = _link.querySelector( '.expandable_cue-close' );
+  var _isAnimating = false;
+  var _clickQueue = 0;
+  var _isExpanded;
+  var _that = this;
+  var _contentHeight = $( _content ).height();
+
+  /**
+   * @returns {Object} The Expandable instance.
+   */
+  function init() {
+    _isExpanded = _dom.classList.contains( 'expandable__expanded' );
+    if ( _isExpanded ) {
+      _setExpandedState();
+    } else {
+      _content.style.display = 'none';
+      _setCollapsedState();
+    }
+    _link.classList.remove( 'u-hide' );
+
+    _target.addEventListener( 'click', _handleClick );
+    return this;
+  }
+
+  /**
+   * @param {number} duration
+   *   The duration of the sliding animation in milliseconds.
+   * @returns {Object} The Expandable instance.
+   */
+  function toggle( duration ) {
+    if ( !_isExpanded ) {
+      this.expand( duration );
+    } else {
+      this.collapse( duration );
+    }
+    return this;
+  }
+
+  /**
+   * @param {number} duration
+   *   The duration of the sliding animation in milliseconds.
+   * @returns {Object} The Expandable instance.
+   */
+  function expand( duration ) {
+    duration = duration || _calculateExpandDuration( _contentHeight );
+
+    this.dispatchEvent( 'beginExpand', { target: _that } );
+    _isAnimating = true;
+    $( _content ).slideDown( {
+      duration: duration,
+      easing: 'easeOutExpo',
+      complete: _expandComplete
+    } );
+
+    return this;
+  }
+
+  /**
+   * @param {number} duration
+   *   The duration of the sliding animation in milliseconds.
+   * @returns {Object} The Expandable instance.
+   */
+  function collapse( duration ) {
+    duration = duration || _calculateCollapseDuration( _contentHeight );
+
+    this.dispatchEvent( 'beginCollapse', { target: _that } );
+    _isAnimating = true;
+    $( _content ).slideUp( {
+      duration: duration,
+      easing: 'easeOutExpo',
+      complete: _collapseComplete
+    } );
+
+    return this;
+  }
+
+  /**
+   * Configure the expanded state appearance.
+   */
+  function _setExpandedState() {
+    _showCollapseCue();
```

Comment:
Hm, that's a good point. I'll see what I can do, it may be due to the order in which things happen, in that depending on the CSS may swap the cue at the wrong point in the animation sequence. I'll check...